packages/devtools_app/test/typed_data_variable_test.dart

```diff
@@ -508,6 +536,80 @@ void main() {
     expect(variable.children.first.displayValue, '[0, -1232.222]',
         skip: !kIsWeb);
   });
+
+  test('Retries getObject calls with no offset/count if error is thrown',
+      () async {
+    final bytes = Uint8List.fromList([0, 1, 2, 3]);
+    final instance = Instance(
+      kind: InstanceKind.kUint8List,
+      id: '123',
+      classRef: null,
+      bytes: base64.encode(bytes.buffer.asUint8List()),
+      identityHashCode: null,
+      length: 4,
+    );
+    final variable = Variable.create(
+      BoundVariable(
+        name: 'test',
+        value: instance,
+        declarationTokenPos: null,
+        scopeEndTokenPos: null,
+        scopeStartTokenPos: null,
+      ),
+      isolateRef,
+    );
+
+    when(manager.service.getObject(any, any, offset: 0, count: 4))
+        .thenThrow('Unrecognized parameters offset / count.');
```

Comment:
how do we know from this test that an exception was thrown?

Reply:
Changed to make it more explicit an exception was thrown (verify we call with the arguments that will cause the exception)